_rules/element-lang-valid-de46e4.md

```diff
@@ -141,22 +184,52 @@ The rule applies to elements within the `body` of a webpage. `html` elements are
 
 ```html
 <html lang="en">
-	<body></body>
+	<body>
+		They wandered into a strange Tiki bar on the edge of the small beach town.
+	</body>
 </html>
 ```
 
 #### Inapplicable Example 2
 
-An empty value for the `lang` attribute is ignored, as the rule only applies to `lang` attributes that are not empty ("").
+An empty `lang` [attribute value][] is ignored, as the rule only applies to `lang` attributes that are not empty (`""`).
+
+```html
+<html>
+	<body>
+		<article lang="">
+			They wandered into a strange Tiki bar on the edge of the small beach town.
+		</article>
+	</body>
+</html>
+```
+
+#### Inapplicable Example 3
+
+The `lang` [attribute value][] has a valid language tag, but does not have a [visible][] [text node][] as a [descendant][] in the [flat tree][].
```

Comment:
Is "hidden" a valid language tag?

Reply:
@carlosapaduarte this [comment](https://github.com/act-rules/act-rules.github.io/pull/1383#discussion_r494234889) and the one prior to it should help.

Reply:
I understand @WilcoFiers point, and I would perhaps side with @EmmaJP that having an invalid value makes the example less clear. 
Nevertheless, the example description cannot state the language tag is valid because it is not :-)